### PR TITLE
Update to work with a deployment of multiple frame processors

### PR DESCRIPTION
**Pull request by GDYendell**
This makes various code changes to work with a deployment of multiple frame processors and makes further improvements to the development environment to run against local odin-data deployments.

### CODE REVIEW

**Review by gilesknap — COMMENTED** [review]
To make this easier I have made a PR into odin-data fastcs-dev branch here: https://github.com/odin-detector/odin-data/pull/341.

Once this is merged then the instructions for setting up a dev environment are simplified to:
```
clone odin-data fastcs-dev branch next door to odin-fastcs
open odin-data in vscode and re-open in container
-------------------------------------------------------
cd ../odin-fastcs
pip install -r dev/requirements.txt
pip install -e '.[dev]'
dev/configure.sh /odin /odin/venv
Hit Enter in right pane of zellij
Debug -> Odin IOC(odin-fastcs)
```
Both fast-cs and odin data can then be modified and tested in this container.

**Comment by gilesknap** [comment]
Hi Gary, do you need any more from me on this PR?

File: README.md [hunk 1]
```diff
@@ -35,8 +35,11 @@ Currently Odin FastCS depends on branches of both odin-control and odin-data, so
...
-fastcs-dev branch of odin-data and build. It is recommended to use the vscode CMake
-configuration to do this.
+fastcs-dev branch of odin-data and build. At DLS it is recommended to use the vscode
+CMake configuration to do this, which can be checked out out from dls-master with
+`git checkout dls-master -- .vscode`. Open odin-data in its own vscode workspace, install
+the recommended extensions and then use `CMake: Delete Cache and Reconfigure` and
+`CMake: Install`.
 
 If you need to run a dev version of any of the applications, stop that process in the
 deployment and run/debug it manually. There is a vscode launch config for an odin server
```
[the remark this answers is not in the document]
Comment (GDYendell) [reply README.md:45]: The pages deployment was not enabled. This should appear on the next tag.

File: README.md [hunk 2]
```diff
@@ -35,8 +35,11 @@ Currently Odin FastCS depends on branches of both odin-control and odin-data, so
 branches are provided in `dev/requirements.txt` for convenience. Make a venv and then
```
[the remark this answers is not in the document]
Comment (GDYendell) [reply README.md]: I have updated it to print a more friendly message, but I want the developer to be intentional in overriding the local deployment.

**Pull request merged**